lib/linux_ccnt/aarch64_cycle_counter.c

```diff
@@ -0,0 +1,86 @@
+/*- C -*- */
+/*
+ *
+ *    Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

Comment:
If you compare this file with other files, you'll see the copyright section has THREE Extra spaces before the first word "Copyright".

Why?